climada/util/coordinates.py

```diff
@@ -801,6 +801,64 @@ def get_region_gridpoints(countries=None, regions=None, resolution=150,
         lat, lon = [ar.ravel() for ar in [lat, lon]]
     return lat, lon
 
+def mapping_point2grid(geometry, ymax, xmin, res):
+    """Given the coordinates of a point, find the index of a grid cell from 
+    a raster into which it falls.
+    
+    Parameters
+    ---------
+    geometry : shapely.geometry.Point object
+        Point which should be evaluated
+    ymax: float
+        coords of top left corner of raster file - y
+    xmin: float
+        coords top left corner of raster file - x
+    res: float or tuple
+        resolution of raster file. Float if res_x=res_y else (res_x, res_y).
+    
+    Returns
+    ------- 
+    col, row : tuple
+        column index and row index in grid matrix where point falls into
+    
+    Raises
+    ------
+    ValueError if Point outside of top left corner of raster
+    """
+    if isinstance(res, tuple):
+        res_x, res_y = (abs(res) for res in res)
+    else:
+        res_x = res_y = abs(res)
+    col = int((geometry.x - xmin) / res_x)
+    row = int((ymax - geometry.y) / res_y)
+    if (col < 0 or row < 0):
+        LOGGER.error('Point not inside grid')
+        raise ValueError  
+    return col, row
```

Comment:
I still have some doubts about this one. 

Is the goal of this function to assign a geographical point (thus, given in a specific coordinate system) to a given grid cell of a raster? Then your function will only work if and only if the raster and the coordinate system are by chance defined in the same coordinate system, which is generally not case right?

Reply:
Sure.
I can put this as a remark in the docstring. 
We have functions in climada to check beforehand whether projections are the same, in case of users needing to check for that.

Reply:
commit 3ce3878 changes geometry to x and y